modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectLayoutLocalServiceTest.java

```diff
@@ -94,7 +94,7 @@ public void testAddObjectLayout() throws Exception {
 			"All required object fields must be associated to the first tab " +
 				"of a default object layout",
 			() -> _objectLayoutLocalService.addObjectLayout(
-				TestPropsValues.getUserId(),
+				RandomTestUtil.randomString(), TestPropsValues.getUserId(),
```

Comment:
It would be nice to assert the externalReferenceCode used to add the ObjectLayout.